CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ Given a version number MAJOR.MINOR.PATCH, increment:
 
 
 ## [Unreleased]
+### Added
+- payer_id and end_to_end_id parameter to BrcodePreview resource
+- cashier_bank_code and description parameter to StaticBrcode resource
+### Changed
+- change nominal_amount and amount parameter to conditionally required to CreditNote resource
```

Comment:
nominal_amount and amount parameter to conditionally required to CreditNote resource